eo-runtime/src/test/java/EOorg/EOeolang/EOarrayEOatTest.java

```diff
@@ -58,4 +60,32 @@ public void pushesAndGetsBack() {
             Matchers.equalTo(txt)
         );
     }
+
+    @Test
+    public void checksNegativeIndex() {
```

Comment:
@Graur We have significant duplication between all three tests. Could we remove it?

Reply:
@volodya-lombrozo I think I got the idea. Let me fix this

Reply:
@volodya-lombrozo Check again please. Does it better now?